### PR TITLE
TRAV-88 - Create DTOs for trip

### PR DESCRIPTION
**Pull request by panayot-marinov**

### CODE REVIEW

**Review by tuechki — DISMISSED** [review]

**Comment by tuechki** [comment]
Some conflicts with dev.

**Comment by panayot-marinov** [comment]
Solved a merge conflict.

File: src/main/java/com/sofia/uni/fmi/travelly/mapper/TripMapper.java [hunk 1]
```diff
@@ -9,29 +10,40 @@ public class TripMapper {
...
-            .users(dto.getUsers())
-            .build();
+                .id(dto.getId())
+                .destination(dto.getDestination())
+                .name(dto.getName())
+                .startDate(dto.getStartDate())
+                .endDate(dto.getEndDate())
+                .budget(dto.getBudget())
+                .interests(dto.getInterests())
+//            .items(dto.getItems())
```
Comment (tuechki) [inline src/main/java/com/sofia/uni/fmi/travelly/mapper/TripMapper.java]: Can you uncomment the code ?

Comment (panayot-marinov) [reply src/main/java/com/sofia/uni/fmi/travelly/mapper/TripMapper.java]: Made a commit for uncommenting

**Review by tuechki — APPROVED** [review]

**Pull request merged**